conda/common/io.py

```diff
@@ -395,16 +395,43 @@ def backdown_thread_pool(max_workers=10):
     """Tries to create an executor with max_workers, but will back down ultimately to a single
     thread of the OS decides you can't have more than one.
     """
+    from concurrent.futures import _base  # These "_" imports are gross, but I don't think there's an alternative  # NOQA
+    from concurrent.futures.thread import _WorkItem
```

Comment:
Need to make sure these imports are ok for python 3.4 and python 3.5.  They obviously are for python 2.7 and 3.6.

Reply:
Confirmed ok for both py34 and py35.